src/main/java/com/amplitude/exception/AmplitudeInvalidAPIKeyException.java

```diff
@@ -0,0 +1,30 @@
+package com.amplitude.exception;
+
+public class AmplitudeInvalidAPIKeyException extends Exception {
+
+    private final int code;
+
+    public AmplitudeInvalidAPIKeyException(int code) {
+        super();
+        this.code = code;
+    }
+
+    public AmplitudeInvalidAPIKeyException(String message, Throwable cause, int code) {
```

Comment:
I guess we don't need to have constructors that we are using

Reply:
I went over some customize examples and tried to make our customize exception covered all the cases. I also think if we do need these constructors or not since we only use one of it. 
Thanks for your suggestion will only leave one constructor.public AmplitudeInvalidAPIKeyException(String message).